Verb with lowercase first letter gets a confusing error

```go
//ftl:subscribe orders
func processOrder(ctx context.Context, event OrderEvent) error {
	return nil
}
```
Build output:
```
info:<modulename>: # main
./main.go:21:27: undefined:<modulename>.processOrder

ftl: error: failed to build module "<modulename>": exit status 1
```
(modulename replaced just for the sake of this ticket)

Adding `ftl:verb` didn't seem to make the error message better either.

Expected: something saying that I must export the function by making the first letter a capital

--- a/go-runtime/compile/schema.go
+++ b/go-runtime/compile/schema.go
@@ -1224,6 +1224,11 @@ func visitFuncDecl(pctx *parseContext, node *ast.FuncDecl) (verb *schema.Verb) {
 		return nil
 	}
 
+	if expVerbName := strcase.ToUpperCamel(node.Name.Name); node.Name.Name != expVerbName {
+		pctx.errors.add(errorf(node, "unexpected verb name %q, did you mean to use %q instead?", node.Name.Name, expVerbName))
+		return nil
+	}
+
 	for _, name := range pctx.nativeNames {
 		if name == node.Name.Name {
 			pctx.errors.add(noEndColumnErrorf(node.Pos(), "duplicate verb name %q", node.Name.Name))

